assets/app/styles/_project-menu.less

```diff
@@ -104,6 +101,12 @@
   }
 }
 
+.navbar-project-menu .bootstrap-select.btn-group .btn .filter-option {
+  max-width: 200px; // Width at 320px
+  min-width: 100%; // Now make menu full width and truncate at mobile
```

Comment:
I'm having trouble following this. If you have min-width: 100%, do you need max-width set at all?